manifests/single-node-openshift/README.md

```diff
@@ -0,0 +1,17 @@
+# Sample Manifests for Single Node OpenShift
+
+This directory includes a collection of manifest files designed to facilitate the deployment of the "testpmd" container within a single-node OpenShift cluster. These manifests have been thoroughly tested and optimized for compatibility with OpenShift version 4.13.6. It's important to note that some of the configuration elements utilized in these manifest files might not be applicable to earlier OpenShift versions. This distinction arises due to the progressive evolution of cri-o features and related provisioning processes in preceding versions.
```

Comment:
Nit pick, but should cri-o be upper cased

Reply:
ack and updated